data/collections/welcome_to_collections.yaml

```diff
@@ -1,4 +1,4 @@
-category: Welcome
+category: Mathematics
```

Comment:
I meant to revert it back. The reason I changed it was because I wanted to see it show up in the library page where results are grouped based on category ('Welcome' is not a valid category).